scalableminds/webknossos
# Datastore Health endpoint sends Ok as text, if Ok

### Steps to test:
- open `/data/health` in a browser, you should see two beautiful letters

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
